content/_guides/faq.md

```diff
@@ -59,6 +59,23 @@ to find when they're connected. There are also some robots, currently
 If you are unable to [connect to the network](/guides/connect) at all, you can
 contact staff by email: <support@libera.chat>.
 
+## What are Global Notices and Wallops, and can I opt out of them?
+
+`Global Notices` are network wide messages that staff can send to everyone on
```

Comment:
```suggestion
**Global Notices** are network wide messages that staff can send to everyone on
```

This isn't an IRC verb or command, but we are defining it as a term, so making it emphasized in some fashion seems good. I think it would read better if the initial usage of it was bold or italicized rather than marked as code. Further usages don't need any special markup in my opinion.